lib/rubocop/config_obsoletion.rb

```diff
@@ -125,6 +125,12 @@ class ConfigObsoletion
         parameters: 'IgnoredMethodPatterns',
         alternative: '`IgnoredMethodPatterns` has been renamed to ' \
                      '`IgnoredPatterns`'
+      },
+      {
+        cops: %w[Performance/Count Performance/Detect],
+        parameters: 'SafeMode',
+        alternative: '`SafeMode` has been removed. ' \
```

Comment:
But it hasn't really been removed yet. :-)

Reply:
Yeah, The `SafeMode` module has not been removed to prevent `NameError`, but the `SafeMode` option has been removed from `Performance/Count` cop and `Performance/Detect` cop.
https://github.com/rubocop-hq/rubocop-performance/pull/72

OTOH, the current released RuboCop Performance 1.14.1 does not include that PR yet. I will release RuboCop Performance early if it is a release blocker for RuboCop core.

Reply:
Got it. Not a big deal. I'm certainly not in a hurry to cut a new RuboCop release.